JPF/utils.py

```diff
@@ -134,10 +134,11 @@ def hpd_csv2sql(description, input_csv_url, sep_char,\
             return
 
         # stream csv into chunked dataframes
-        log.info("Streaming CSV from {} ...".format(input_csv_url))
+        log.info("Streaming {} CSV fields from {} ...".format(
+            len(keep_cols), input_csv_url))
 
         csv_chunks = pd.read_csv(input_csv_url, sep=sep_char, dtype=dtype_dict,\
-                encoding='utf8', chunksize=csv_chunk_size)
+                encoding='utf8', chunksize=csv_chunk_size, usecols=keep_cols)
```

Comment:
awesome .. i should have read those docs weeks ago .. at least I made it easy to utilize the usecols ... does it matter that my keep_cols is after lowered slugified cols?